README.md

```diff
@@ -1,3 +1,8 @@
+---

```

Comment:
What is this for? (Note, this renders in markdown).

Reply:
This is used in website build. @fadeev came up with this convention, I don't know that we can prevent flags to be rendered

Reply:
we use that in the website build. I'll see with @fadeev if we can change convention to

```
<!-- 
--- 
parent:
  order: false
--- 
-->
```

so that it does not render in markdown

Reply:
`<!--` won’t work out of the box, because VuePress parses files for `---` (this is hard-coded into VP) and then passes them to `gray-matter` JS-lib for further conversion into JSON.

https://github.com/vuejs/vuepress/blob/master/packages/@vuepress/shared-utils/src/parseVueFrontmatter.ts#L5

Of course, we can comment frontmatter out and strip `<!--` and `-->` at build even with something like sed `s/<!--//g` . That being said, I think we should think about maintainability of all this before we add any more custom things to our markdown files. I’m not saying we shouldn’t (I actually like extending stuff), I just think we should think about it, or we’ll to end up with a hodgepodge of custom syntax parsed with hacky shell-scripts

Reply:
We should add a link on the `docs/README.md` to the rendered documentation on the website. I think that suffices (at least for me).

Reply:
Great @fedekunze. Will do so as soon as this PR is merged. 

We will think of ways to improve readability of github in later upgrades